wallet/src/repository/error.rs

```diff
@@ -43,6 +44,8 @@ pub enum Error {
     HashParseError(#[cause] types::HashParseError),
     #[fail(display = "failed because wallet is still syncing: {}", _0)]
     StillSyncing(String),
+    #[fail(display = "failed creating a transaction: {}", _0)]
+    TransactionCreation(#[cause] TransactionError),
```

Comment:
```suggestion
    TransactionCreationError(#[cause] TransactionError),
```

Reply:
Other repository errors has not the Error sufix

Reply:
Clippy sometimes complains about using an `Error` suffix, but on the other hand there is a `HashParseError` there... In my opinion you can leave it like this

Reply:
I agree with @tmpolaczyk